Library/Formula/glibc.rb

```diff
@@ -19,6 +21,8 @@ def install
         "--prefix=#{prefix}",
         "--enable-obsolete-rpc",
         "--without-selinux"] # Fix error: selinux/selinux.h: No such file or directory
+      args << "--enable-kernel=" +
+        `uname -r | cut -d. -f1,2,3 | cut -d- -f1` if build.with? "current-kernel"
```

Comment:
What does `uname -r` look like on a non-distribution kernel? If it's simply `3.7.5`, that will also work with this patch.